avocado/utils/disk.py

```diff
@@ -25,10 +25,17 @@
 import os
 import json
 import re
+import string
 
 from . import process
 
 
+class DiskUtilsError(Exception):
```

Comment:
This is an excellent intention, but I suggest you use a more specific exception, like `OSError`, for example.

Reply:
OSError is generic, too.

Reply:
@beraldoleal, I agree, but it, at least, covers a more specific set of exceptions.

Reply:
@willianrampazzo idea here is it catch Exceptions for  utility, right now it seems  OSError but idea is that Exceptions can be reused for any exception from this utility as I am planning to  add more library will use this, please let me your thought

Reply:
IMO, `DiskUtilsError` is better than an `OSError`, because tells the user that the error was in our library. But, I might be missing something.

Reply:
@beraldoleal I also agree with this @willianrampazzo please let us know your thought so we can have  V2

Reply:
My initial comment was about changing `class DiskUtilsError(Exception)` by `class DiskUtilsError(OSError)` as `OSError` is more specific to what is been covered here, and not changing the `DiskUtilsError` by `OSError`.

But I don't have problems keeping the way it is now, it was just a suggestion to restrict the kind of exceptions covered by `DiskUtilsError`.

Reply:
@willianrampazzo sure but I feel we need to have DiskUtilsError having broader   Exception so we need to change it on future but when we raise it in utility we can narrow down there like in OSError case we have to expect in that utility how that sound

Reply:
Okay, I got it. Let's go with `Exception`.